vermouth/tests/test_molecule.py

```diff
@@ -1229,7 +1229,13 @@ def test_interaction_sort(interactions, expected):
     assert vermouth.molecule.Molecule.sort_interactions(interactions) == expected
 
 
-@hypothesis.given(moltype=st.one_of(st.none(), st.text()), mol=random_molecule())
+@hypothesis.given(
+    moltype=st.one_of(
+        st.none(),
+        st.text(st.characters(blacklist_categories=('C', 'Z')), min_size=1)
```

Comment:
Maybe an example case with a character from class C and Z just to know how things explode or not? I would not fight for it, though. If you are using a bell character as an atom name you are looking for problems and you deserve them.

Reply:
I had false test failures (because of how I test `molecule.__str__`) where an interaction type is named "0 a" with 0 interactions, which is in "0 atoms", resulting in test failure. It doesn't blow up perse. 
Or a molecule name matching an empty interaction type, something along those lines. 

Easier to sanitize the input a bit than figure out and `asssume` out all possible false failure modes.